README.md

```diff
@@ -1,126 +1,42 @@
-# dbildungs-iam-keycloak
+# Keycloak Container Image
 
-dbildungs-iam-keycloak is a customized [`Keycloak`](https://github.com/keycloak/keycloak) docker image for the [dBildungscloud-iam-server](https://github.com/dBildungsplattform/dbildungs-iam-server).
+This repository provides a Docker container image for running Keycloak. The container is built on Debian 12 and includes OpenJDK 17. Building different variants based on customer/project needs is supported.
 
-Images for local development or production are build for each Git tag via GitHub Actions.
+## Configuration Options
 
-## Automtic image build and scan for development
-On every push to a branch (except main) a GitHub Workflow builds, uploads (to GHCR) and scans the image (with trivy). The image is tagged with the ticket extracted from the branch name. The trivy scan results are available under Security -> Code scanning -> Filter: "is:open branch:\<branch\>".
+### Build Arguments
 
-After a merge to main a GitHub Workflow builds, uploads (to GHCR) and scans the image (with trivy). The image is tagged with the short commit hash and "latest". The trivy scan results are available under Security -> Code scanning.
+* **`KEYCLOAK_VERSION`**: Version of Keycloak to be downloaded and installed. Default is `26.2.5`.  
+  * Example: `--build-arg KEYCLOAK_VERSION=27.0.0`
 
-The latest image on GHCR is scanned with trivy every night.
+* **`KEYCLOAK_VARIANT`**: Defines the variant of Keycloak to be used. This refers to a subdirectory in `variants/` which contains a env file and supports adding more files (like themes) to keycloak.
+  * `generic` (default): A general-purpose variant suitable for most use cases.  
+  * Example: `--build-arg KEYCLOAK_VARIANT=custom`
 
-## Manual build for local development image
+More basic Keycloak configurations can be found at `variants/base/env`.
 
-Following steps are intent to build the container for local development purpose.
+### Environment Variables
```

Comment:
Please state that env vars from base can be overwritten by env vars in variants.